examples/text_summarization/extractive_summarization_cnndm_aml_distributed.ipynb

```diff
@@ -0,0 +1,795 @@
+      "Failure while loading azureml_run_type_providers. Failed to load entrypoint azureml.StepRun = azureml.pipeline.core:StepRun._from_dto with exception (azureml-core 1.0.83 (/dadendev/anaconda3/envs/cm3/lib/python3.6/site-packages), Requirement.parse('azureml-core==1.0.57.*')).\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "SDK version: 1.0.83\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "from tempfile import TemporaryDirectory\n",
+    "import torch\n",
+    "\n",
+    "nlp_path = os.path.abspath('../../')\n",
+    "if nlp_path not in sys.path:\n",
+    "    sys.path.insert(0, nlp_path)\n",
+    "    \n",
+    "from utils_nlp.azureml import azureml_utils\n",
+    "\n",
+    "# Check core SDK version number\n",
+    "import azureml.core\n",
+    "print(\"SDK version:\", azureml.core.VERSION)\n",
+    "\n",
+    "from azureml.core import Experiment, Workspace, Run\n",
+    "from azureml.core.compute import ComputeTarget, AmlCompute\n",
+    "from azureml.core.compute_target import ComputeTargetException\n",
+    "from azureml.train.dnn import PyTorch\n",
+    "from azureml.widgets import RunDetails\n",
+    "from azureml.train.dnn import Nccl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Replace the following constants with your own \n",
+    "WORKSPACE_NAME = \"daden1amlws\"\n",
+    "SUBSRIPTION_ID = \"9086b59a-02d7-4687-b3fd-e39fa5e0fd9b\" \n",
+    "RESOURCE_GROUP = \"daden1aml\"\n",
+    "LOCATION = \"eastus2\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the workspace using the specified parameters\n",
+    "ws = Workspace.create(name = WORKSPACE_NAME,\n",
+    "                      subscription_id = SUBSRIPTION_ID,\n",
+    "                      resource_group = RESOURCE_GROUP, \n",
+    "                      location = LOCATION,\n",
+    "                      create_resource_group = False,\n",
+    "                      exist_ok = True)\n",
+    "ws.get_details()\n",
+    "\n",
+    "# write the details of the workspace to a configuration file to the notebook library\n",
+    "ws.write_config()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "setup() is now deprecated. Instead, please use create() to create a new workspace, or get()/from_config() to retrieve an existing one\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Workspace name: daden1amlws\n",
+      "Workspace region: eastus2\n",
+      "Subscription id: 9086b59a-02d7-4687-b3fd-e39fa5e0fd9b\n",
```

Comment:
you might want to clear sensitive info (subscription ID, etc...)

Reply:
will do